definition/data/sheets/CaseEvent.json

```
@@ -78,5 +78,6 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "ReviewJudgeComments", "Name": "Reviewed Judge's comments", "Description": "Reviewed Judge's comments", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutURLAboutToSubmitEvent": 0},
   {"CaseTypeID": "MoneyClaimCase", "ID": "ManageDocuments", "Name": "Manage Documents", "Description": "Manage Documents", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "ClaimantResponseReceiptUpload", "Name": "Claimant response uploaded", "Description": "Claimant response uploaded", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "ChangeContactDetails", "Name": "Change contact details", "Description": "Change the contact details for any party.", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": 0, "EndButtonLabel": "Submit Changes" }
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ChangeContactDetails", "Name": "Change contact details", "Description": "Change the contact details for any party.", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": 0, "EndButtonLabel": "Submit Changes" },
+  {"CaseTypeID": "MoneyClaimCase", "ID": "UpdateBulkPrintDetails", "Name": "Bulk print Request details", "Description": "add Bulk print details for the request.", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public"}
```

Is the capitalisation on the name and description correct here, i.e 
```suggestion
  {"CaseTypeID": "MoneyClaimCase", "ID": "UpdateBulkPrintDetails", "Name": "Bulk print request details", "Description": "Add Bulk print details for the request.", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public"}
```

---

done